test-requirements.in

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env pip-compile -q --allow-unsafe --output-file=test-requirements.txt
+# Avoid using --generate-hashes as it breaks pip install from tox with:
+# ERROR: In --require-hashes mode, all requirements must have their versions pinned with ==. These do not:
```

Comment:
Uh, I know what to do but that's for another PR

Reply:
That's the spirit, lets progress even if some parts may not be ideal. As long there is no clear regression in a change, there is no need to delay it even further, especially for long standing ones.